CMakeLists.txt

```diff
@@ -15,16 +15,24 @@
 project(deepstate)
 cmake_minimum_required(VERSION 2.8)
 
-if (NOT DEFINED BUILD_LIBFUZZER AND DEFINED ENV{BUILD_LIBFUZZER}) 
-  set(BUILD_LIBFUZZER "$ENV{BUILD_LIBFUZZER}") 
+if (NOT DEFINED BUILD_LIBFUZZER AND DEFINED ENV{BUILD_LIBFUZZER})
```

Comment:
We should prefix these variables with `DEEPSTATE_` and then turn them into `option()`s